src/react/useAtomValue.ts

```diff
@@ -94,7 +94,6 @@ export function useAtomValue<Value>(atom: Atom<Value>, options?: Options) {
       }
       rerender()
     })
-    rerender()
```

Comment:
This is required. I wonder if we could make a contrived test to cover it.

(I thought there was a rejected PR or something to refer, but can't find it now.)

Reply:
yeah, maybe we should add more test. but the rerender should be optimized.

Reply:
Did you have a real problem in your usage? If you have, memoization might mitigate the issue for the meantime.